src/app/file-manager/transferables-grid/transferables-grid.component.ts

```diff
@@ -15,6 +15,11 @@ import { FilesDatabase } from '../dbstate/files-database';
 import { LimitTransferablesService } from '../services/limit-transferables.service';
 import { Type } from '@app/file-manager/models/type';
 import { ItemStatus } from '@app/file-manager/models/item-status';
+import { NgxSpinnerComponent } from 'ngx-spinner';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { modelGroupProvider } from '@angular/forms/src/directives/ng_model_group';
```

Comment:
Some of these imports don't appear to be used: `NgxSpinnerComponent`, `BehaviorSubject`, `modelGroupProvider`.